jena-integration-tests/src/test/java/org/apache/jena/test/TestSettings.java

```diff
@@ -30,7 +30,8 @@
 
     @Test
     public void setting_RDFstar_fastpath() {
-        assertTrue(org.apache.jena.sparql.engine.iterator.RX.DATAPATH);
+        assertTrue(org.apache.jena.sparql.engine.main.solver.SolverRX3.DATAPATH);
+        assertTrue(org.apache.jena.sparql.engine.main.solver.SolverRX4.DATAPATH);
```

Comment:
What is this testing?

Reply:
Javadoc for SolverRX3:
```
    /**
     * This constant is not public API. It is exposed only so integration testing can
     * check the value for a release build.
     */
```
i.e. making sure no accidents.